test/functions_linearalgebra.jl

```diff
@@ -199,3 +199,15 @@ end
         NamedDimsArray{(:A, :B)}(r1), NamedDimsArray{(:NotA,)}(b)
     )
 end
+
+@testset "isposdef" begin
+    A = [1 2; 2 50]
+
+    @test isposdef(A)
```

Comment:
```suggestion
    @assert isposdef(A)
```
When it can't be fixed by changing something `src/`, I think it is better to use `@assert`.
It is just an assumption of the tests

Reply:
Yeah, I guess that makes sense.